core/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -198,6 +200,7 @@
 public class InternalEngineTests extends ESTestCase {
 
     protected final ShardId shardId = new ShardId(new Index("index", "_na_"), 0);
+    protected final AllocationId allocationId = AllocationId.newInitializing();
```

Comment:
maybe just store the string here, i.e. 
`protected final String allocationId = AllocationId.newInitializing().getId();`

Reply:
The actual allocation ID instance is needed in one place. 😐

Reply:
ok 🆗